Positioning2/style.css

```
@@ -0,0 +1,66 @@
+.parent {
+    background-color: rgb(230, 10, 10)7;
+    display: flex;
+    width: 555px;
+    height: 191px;
+
+    border-style: dotted;
+    border-color: rgb(87, 67, 93);
+    border-radius: 10px;
+    border-width: 5px;
+
+    top: 75px;
+
+    text-align: center;
+}
+
+.parent {
+    background-color: rgb(168, 176, 167);
+    top: 0;
+    left: 50%;
+}
+
+h2 {
+    display: flex;
+    justify-content: center;
+    padding-top: 20px;
+}
+
+.turquoise h2{  
+    background-color: rgb(64, 184, 197);
+    width: 180px;
+    height: 120px;
+    border-style: dashed;
+    border-color: rgb(87, 67, 93);
+    border-radius: 10px;
+    margin: 0;
+    position: absolute;
+    z-index: 1;
+}
```

Aici nu trebuie sa stilizam si h2-ul, iar patratul turcuaz trebuie pus in coltul de stanga sus fiind de ajuns sa-i dam `top: 0;` (fara margin);

```suggestion
.turquoise {
  background-color: rgb(64, 184, 197);
  width: 180px;
  height: 120px;
  border-style: dashed;
  border-color: rgb(87, 67, 93);
  border-radius: 10px;
  position: absolute;
  top: 0;
  z-index: 1;
}
```